dockerfiles/entrypoints/celery-beat.sh

```diff
@@ -4,6 +4,11 @@
 
 CMD="python3 -m celery -A ${CELERY_APP_NAME}.worker beat -l ${CELERY_LOG_LEVEL}"
 
+# Not sure why, but the db becomes corrupted at some point
+# and make the startup of this process to fail.
+# Removing it makes Celery to re-create it.
+rm celerybeat-schedule.db
```

Comment:
maybe `-f`? It doesn't exist this will fail

Reply:
Yes. Makes sense 👍

Reply:
```suggestion
rm -f celerybeat-schedule.db
```